tests/cli/commands/test_sync_perm_command.py

```diff
@@ -51,6 +56,7 @@ def test_cli_sync_perm(self, dagbag_mock, mock_cached_app):
         assert appbuilder.sm.sync_roles.call_count == 1
 
         dagbag_mock.assert_called_once_with(read_dags_from_db=True)
+        collect_dags_from_db_mock.assert_called_once_with()
```

Comment:
nit: better to use `.assert_called_once()`?

Reply:
Does not matter much tbh, `assert_called_once_with()` is a superset of `assert_called_once`, in this case it just checks that it is called with no args.